### DISCUSSION TITLE
mi practica de pastelería html me

### DISCUSSION THREAD

**Discussion by junae67** [Launch-X-Latam/MisionFrontEnd#381] [General]
esforcé mucho aun que al final no puede implementar el css en todo no tengo tano nivel pero hay sigo aprendiendo me gustaría escuchar que opinan 

 la página  https://pasteleriapractica.herokuapp.com/Pasteleria.html
mi repositorio https://github.com/junae67/Pagina
la iré mejorando con el tiempo a medida que aprenda

**Comment by OscarJG20** [comment]
Hola!
Me gustó, y muy bien que considerarás el responsive desde el principio.

2 cuestiones que noté en tu imagen de portada.
El botón de comprar es un <button>, en caso de que sea una redirección a otra página debería ser un <a>, en cualquiera de los 2 casos para darle más interactividad, en los estilos del botón podrías poner "cursos: pointer;", de esa manera al pasar el mouse se ve más claro que ocurrirá algo al presionarlo.

Otra cosa con  la imagen de portada, es estético (no es necesario), podrías poner el css de la imagen "background-attachment: fixed;" la propiedad lo que hace es mantener fija la imagen mientras haces scroll, genera un efecto curioso.

**Reply by junae67** [reply]
gracias por opinar me hace sentir  bien y me anima  tomare todo lo que me contentaste para seguir mejorando la pagina
